backend/lib/endpoints/schema/organizations.js

```diff
@@ -26,22 +50,17 @@ const getOrganizationsSchema = {
 
 const updateOrganizationSchema = {
   body: strictSchema()
-    .prop("address", S.string())
-    .prop("androidUrl", S.string())
-    .prop("description", S.string())
-    .prop("email", S.string())
-    .prop("global", S.boolean())
-    .prop("industry", S.string())
-    .prop("iosUrl", S.string())
-    .prop("language", S.string())
-    .prop("linkedinUrl", S.string())
-    .prop("location", S.string())
-    .prop("name", S.string())
-    .prop("needs", S.array().maxItems(10).items(S.string()))
-    .prop("ownerId", S.string())
-    .prop("twitterUrl", S.string())
-    .prop("type", S.string())
-    .prop("website", S.string()),
+    .prop("about", organization.about)
+    .prop("email", organization.email)
```

Comment:
same thing, can you just update an email address? 
Or maybe it can only be updated when it's verified